File: sources/blog/offline/apps/post/urls.js
require('doff.conf.urls', '*');
require('blog.blog.views');

var urlpatterns = patterns('',
    ['^add_tag/$', 'blog.blog.views.add_tag'],
    ['^remove_tag/([A-Za-z0-9-]+)/$', 'blog.blog.views.remove_tag'],
    ['^add_post/$', views.add_post],
    ['^remove_post/([A-Za-z0-9-]+)/$', views.remove_post]
);

publish({ 
    urlpatterns: urlpatterns 
});

File: sources/blog/post/urls.py
# -*- coding: utf-8 -*-


from django.conf.urls.defaults import *
from blog.post import views
    
urlpatterns = patterns('django.views.generic.simple', 
    (r'^add_tag/$', views.add_tag),
    (r'^remove_tag/$', views.remove_tag),
    (r'^add_post/$', views.add_post),
    (r'^remove_post/$', views.remove_post)
)